hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/KafkaOffsetGen.java

```diff
@@ -165,6 +169,7 @@ public KafkaOffsetGen(TypedProperties props) {
     }
     DataSourceUtils.checkRequiredProperties(props, Collections.singletonList(Config.KAFKA_TOPIC_NAME));
     topicName = props.getString(Config.KAFKA_TOPIC_NAME);
+    kafkaCheckpointTimestamp = props.getString(Config.KAFKA_CHECKPOINT_TIMESTAMP);
```

Comment:
if the value of `Config.KAFKA_CHECKPOINT_TIMESTAMP`  does not exist, Exception will be thrown, this is not expected when the user want to use checkpoint by providing offsets

Reply:
yes， i will correct